runtime/lua/vim/treesitter/dev.lua

```diff
@@ -334,7 +336,13 @@ function M.inspect_tree(opts)
 
   -- window id for source buffer
   local win = api.nvim_get_current_win()
-  local treeview = assert(TSTreeView:new(buf, opts.lang))
+  local treeview, err = TSTreeView:new(buf, opts.lang)
+  if not treeview and err and err:match('no parser for lang') then
```

Comment:
If `err` is set, `treeview` must be `nil`:
```suggestion
  if err and err:match('no parser for lang') then
```